### PR TITLE
Translating on Polish

### PR DESCRIPTION
**Pull request by RozanovK**

### CODE REVIEW

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+\myparagraph{GCC}
+
+Skompilujmy przykład w GCC 4.8.1 dla ARM64:
+
+\lstinputlisting[numbers=left,label=hw_ARM64_GCC,caption=\NonOptimizing GCC 4.8.1 + objdump,style=customasmARM]{patterns/01_helloworld/ARM/hw.lst}
+
+W ARM64 nie ma trybów Thumb i Thumb-2, tylko ARM, dlatego tu korzystamy tylko z 32-bitowych instrukcji.
+
+Jest tu dwa razy więcej rejestrów: \myref{ARM64_GPRs}.
+64-bitowe rejsetry mają prefiks
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: typo: `rejestry`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+W ARM64 nie ma trybów Thumb i Thumb-2, tylko ARM, dlatego tu korzystamy tylko z 32-bitowych instrukcji.
+
+Jest tu dwa razy więcej rejestrów: \myref{ARM64_GPRs}.
+64-bitowe rejsetry mają prefiks 
+\TT{X-}, a ich 32-bitowe części --- \TT{W-}.
+
+\myindex{ARM!\Instructions!STP}
+Instrukcja \TT{STP} (\IT{Store Pair}) 
+odkłada na stos od razu 2 rejestry: \RegX{29} i \RegX{30}.
+Oczywiście, ta instrukcja może zapisać tę parę rejestrów gdziekolwiek w pamięci, ale tu jest ukazany rejestr \ac{SP}, także ta para jest odkładana na stos.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: `s/ukazany/wskazany/`?

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Rejestry w ARM64 są 64-bitowe, każdy o długości 8 bajt, dlatego do przechowywania 2 rejestrów potrzeba 16 bajtów.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: `s/bajt/bajtów/`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+Wykrzyknik (``!'') po operandzie oznacza, że najpierw od \ac{SP} będzie odjęte 16 i dopiero po tej czynności wartości z obu rejestrów będą odłożone na stos.
+
+Na to się mówi \IT{pre-index}.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: slang, `Jest nazywane; Nazywa się`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+Więcej o róznicy między \IT{post-index} a \IT{pre-index} 
+można znaleźć tu: \myref{ARM_postindex_vs_preindex}.
+
+W ten sposób, posługując się terminologią x86, pierwsza instrukcja~--- jest analogiczna do \TT{PUSH X29} i \TT{PUSH X30}.
+\RegX{29} w ARM64 jest wykorzystywane jako \ac{FP}, a \RegX{30} 
+jako \ac{LR}, dlatego one są zapisywane w prologu funkcji.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: syntax, `są one`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+Druga instrukcja kopiuje \ac{SP} do \RegX{29} (lub \ac{FP}).
+Jest to niezbędne do ustawienia stack frame funkcji.
+
+\label{pointers_ADRP_and_ADD}
+\myindex{ARM!\Instructions!ADRP/ADD pair}
+Instrukcje \TT{ADRP} i \ADD są potrzebne do sformowania adresu linii \q{Hello!} w rejestrze \RegX{0},
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: `sformowania`? maybe: `skonstruowania`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+jako \ac{LR}, dlatego one są zapisywane w prologu funkcji.
+
+Druga instrukcja kopiuje \ac{SP} do \RegX{29} (lub \ac{FP}).
+Jest to niezbędne do ustawienia stack frame funkcji.
+
+\label{pointers_ADRP_and_ADD}
+\myindex{ARM!\Instructions!ADRP/ADD pair}
+Instrukcje \TT{ADRP} i \ADD są potrzebne do sformowania adresu linii \q{Hello!} w rejestrze \RegX{0}, 
+jako że pierwszy argument f-cji jest przekazywany przez ten rejestr.
+Ale w ARM nie ma instrukcji, za pomocą których można zapisać do rejestru dużą liczbę
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: Syntax: do not start the sentence with `Ale`. Consider: `Jednakże`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+Instrukcja \MOV zapisuje 0 do \RegW{0}. 
+\RegW{0} to młodsze 32 bity 64-bitowego rejestru \RegX{0}:
+
+\input{ARM_X0_register}
+
+Wynik funkcji jest zwracany przez \RegX{0}, i \main zwraca 0.
+
+Dlaczego 32-bitowa część?
+Dletgo że w ARM64, jak i w x86-64, typ \Tint zostawili 32-bitowym, dla kompatybilności.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: typo: `Dlatego`; could be a little more formal

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+Odpowiednio, jako że funkcja zwraca 32-bitowy \Tint, to gtrzeba wypełnić tylko młodsze 32 bity 64-bitowego rejestru \RegX{0}.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: typo: `trzeba`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+\begin{lstlisting}[caption=\NonOptimizing GCC 4.8.1 + objdump]
+  4005a4:       d2800000        mov     x0, #0x0      // #0
+\end{lstlisting}
+
+\myindex{ARM!\Instructions!LDP}
+Następnie za pomocą instrukcji \INS{LDP} (\IT{Load Pair}) są przywracane rejestry \RegX{29} i \RegX{30}.
+
+Wykrzyknika po instrukcji brak. To oznacza, że najpierw wartości są zdejmowane ze stosu, i dopiero po tej czynności \ac{SP} jest zwiększane o 16.
+
+Na to się mówi \IT{post-index}.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: ibid

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+
+\myindex{ARM!\Instructions!RET}
+W ARM64 pojawia się nowa instrukcja: \RET. 
+Ona działa tak samo jak i \INS{BX LR}, ale zawiera bit ,
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: style: `s/Ona działa/Działa/`; `s/i //`

File: patterns/01_helloworld/ARM/ARM64_PL.tex [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+który podpowiada procesorowi, że jest to wyjście z f-cji, a nie zwykłe przejście, żeby procesor mogł zoptymalizować tę instrukcję.
```
Comment (AwesomePatrol) [inline patterns/01_helloworld/ARM/ARM64_PL.tex]: typo: `mógł`

**Comment by dennis714** [comment]
Thanks a lot.
I think, you can write your comments to other Polish translators right in the source code, commented.
Each comment in TeX/LaTeX is started with percent (%) sign.

**Pull request merged**